### PR TITLE
AWS SNS - Send SMS

### PR DESCRIPTION
**Pull request by philippegabert**
Send SMS using Amazon SNS

### CODE REVIEW

**Comment by mellistibco** [comment]
@philippegabert can you remove the bme280 stream from this pr?

**Comment by philippegabert** [comment]
@mellistibco All done !

File: activity/awssns/activity.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Amazon SNS - SMS",
+  "version": "0.0.1",
+  "type": "flogo:activity",
+  "description": "Send SMS using Amazon Web Services Simple Notification Services",
+  "author": "Phlippe GABERT <pgabert@tibco.com>",
+  "ref": "github.com/philippegabert/flogo-contrib/blob/awssns/activity/awssns",
```
Comment (mellistibco) [inline activity/awssns/activity.json]: This should be updated to github.com/TIBCOSoftware/flogo-contrib/activity/awssns

File: activity/awssns/activity.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	ConfAWSDefaultRegion   = "AWS_DEFAULT_REGION"
+	ConfSMSType            = "SMS_TYPE"
+	ConfSMSFrom            = "SMS_FROM"
+	ConfSMSTo              = "SMS_TO"
+	ConfSMSMessage         = "SMS_MESSAGE"
+	OUTMessageID           = "MESSAGE_ID"
+)
+
+// log is the default package logger
+var log = logger.GetLogger("activity-tibco-configreader")
```
Comment (mellistibco) [inline activity/awssns/activity.go]: rename to indicate sns

File: activity/awssns/activity.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  "homepage": "https://github.com/philippegabert/flogo-contrib/blob/awssns/activity/awssns",
```
Comment (mellistibco) [inline activity/awssns/activity.json]: This should also be changed to github.com/TIBCOSoftware...

File: activity/awssns/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+# Amazon SNS - SMS
+This activity sends SMS using Amazon Simple Notification Services (SNS).
+
+
+## Installation
+
+```bash
+flogo add activity github.com/philippegabert/flogo-contrib/activity/awssns
```
Comment (mellistibco) [inline activity/awssns/README.md]: update to reflect TIBCOSoftware/flogo-contrib

**Comment by philippegabert** [comment]
@mellistibco Done !

**Comment by vijaynalawade** [comment]
@mellistibco @philippegabert : Sorry. I missed this. 
Looks great. Just below suggestion about **smsType** configuration:
`{
      "name": "smsType",
      "type": "string",
      "allowed" : ["Promotional", "Transactional"],
      "value": "Promotional"
  }`
This will set default value to **Promotional**.
Following code then can be further simplified:
`if context.GetInput(ConfSMSType) == nil {
 		log.Info("SMS Type not set. Using default value 'Promotional'")
 		SMSType = "Promotional"
 	} else {
 		SMSType = context.GetInput(ConfSMSType).(string)
 	}`
**To:**
   `SMSType := context.GetInput(ConfSMSType).(string)
   `

**Comment by philippegabert** [comment]
Hi, 

see #202 for updates

Philippe

**Pull request merged**